### PR TITLE
New package: curlpp-0.7.3

### PR DESCRIPTION
**Pull request by asafo**

### CODE REVIEW

File: srcpkgs/curlpp/template [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+
+pre_configure()
+{
+    ./autogen.sh
+}
+
+do_configure()
+{
+    export CPPFLAGS=" -Wno-error=unused-function -O2 -pipe"
```
Comment (xtraeme) [inline srcpkgs/curlpp/template]: You can remove `do_configure` completely and set `CPPFLAGS="..."` in the top level block.

File: srcpkgs/curlpp/template [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+maintainer="Asaf Ohayon <asaf@sysbind.co.il>"
+license="MIT"
+homepage="http://www.curlpp.org/"
+distfiles="https://github.com/jpbarrette/curlpp/archive/v${version}.tar.gz"
+checksum=b72093f221a9e2d0f7ce0bd0f846587835e01607a7bb0f106ff4317a8c30a81c
+
+
+pre_configure()
+{
+    ./autogen.sh
```
Comment (xtraeme) [inline srcpkgs/curlpp/template]: Please keep the common style used in all other packages:

```
func() {
   ....
}
```

File: srcpkgs/curlpp/template [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Template file for 'curlpp'
+pkgname=curlpp
+version=0.7.3
+revision=1
+build_style=gnu-configure
+hostmakedepends="automake libtool"
+makedepends="libcurl-devel boost-devel"
+short_desc="cURLpp is a C++ wrapper for libcURL"
+maintainer="Asaf Ohayon <asaf@sysbind.co.il>"
+license="MIT"
```
Comment (xtraeme) [inline srcpkgs/curlpp/template]: `MIT` license needs to be installed via `post_install()`, i.e:

```
post_install() {
    vlicense LICENSE
}
```

File: srcpkgs/curlpp/template (hunk 2 above)
Comment (asafo) [inline srcpkgs/curlpp/template]: Need to make a new pull request for this fix ?

File: srcpkgs/curlpp/template [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# Template file for 'curlpp'
+pkgname=curlpp
+version=0.7.3
+revision=1
+build_style=gnu-configure
+hostmakedepends="automake libtool"
+makedepends="libcurl-devel boost-devel"
+short_desc="cURLpp is a C++ wrapper for libcURL"
```
Comment (xtraeme) [inline srcpkgs/curlpp/template]: Please remove `cURLpp is a`; it's redundant.

File: srcpkgs/curlpp/template (hunk 2 above)
Comment (Vaelatern) [inline srcpkgs/curlpp/template]: Amend the commit and force push.

**Comment by asafo** [comment]
So how to make into the tree ?

File: srcpkgs/curlpp/template [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+hostmakedepends="automake libtool"
+makedepends="libcurl-devel boost-devel"
+short_desc="C++ wrapper for libcURL"
+maintainer="Asaf Ohayon <asaf@sysbind.co.il>"
+license="MIT"
+homepage="http://www.curlpp.org/"
+distfiles="https://github.com/jpbarrette/curlpp/archive/v${version}.tar.gz"
+checksum=b72093f221a9e2d0f7ce0bd0f846587835e01607a7bb0f106ff4317a8c30a81c
+
+CPPFLAGS=" -Wno-error=unused-function -O2 -pipe"
```
Comment (Vaelatern) [inline srcpkgs/curlpp/template]: I'm not sure as to the effects of this. Can you use `+=` and why did you use `-O2 -pipe`? Don't we already have these?

Comment (chneukirchen) [inline srcpkgs/curlpp/template]: On toplevel, `=` is fine. But just the `-W` should be enough indeed.

**Pull request closed without merging**